src/api.ts

```diff
@@ -0,0 +1,16 @@
+const API_URL = 'https://jsonplaceholder.typicode.com';
+
+const getItems = async<T>(url: string): Promise<T[]> => {
+  const response = await fetch(API_URL + url);
```

Comment:
We can use template literals to avoid some bugs with this method: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals